src/stream/src/executor/lookup/impl_.rs

```diff
@@ -72,24 +73,6 @@ pub struct LookupExecutorParams<S: StateStore> {
     /// of this `arrangement_order_rules`.
     pub arrangement_order_rules: Vec<ColumnOrder>,
 
-    /// Primary key indices of the lookup result (after reordering).
-    ///
-    /// [`LookupExecutor`] will lookup a row from the stream using the join key in the arrangement.
-    /// Therefore, the output of the [`LookupExecutor`] will be:
-    ///
-    /// ```plain
-    /// | stream columns | arrangement columns |
-    /// ```
-    ///
-    /// ... and will be reordered by `output_column_reorder_idx`.
-    ///
-    /// The optimizer should select pk with pk of the stream columns, and pk of the original
-    /// materialized view (upstream of arrangement).
```

Comment:
Maybe we can keep these comments in the struct's doc, or the info field

Reply:
Looks like the comment for `column_mapping` field already covers most of this.